CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 
 * Regression: Unhelpful exception message when setting up an indexer with `SetupProperty` (@stakx, #823)
 
+#### Changed
+
+* Improved performance for `Mock.Of<T>` and `mock.SetupAllProperties()` as the latter now performs property setups just-in-time, instead of as an ahead-of-time batch operation. (@vanashimko, #826)
```

Comment:
Please add under a new `## Unreleased` heading at the top of the changelog;  we're not changing an already published version, after all. :)

Reply:
I've placed this under existing

```
## Unreleased

#### Fixed

* Regression: Unhelpful exception message when setting up an indexer with `SetupProperty` (@stakx, #823)

#### Changed

* Improved performance for `Mock.Of<T>` and `mock.SetupAllProperties()` as the latter now performs property setups just-in-time, instead of as an ahead-of-time batch operation. (@vanashimko, #826)
```

Should I add new (the second) `## Unreleased` for my one?

Reply:
No. You're apparently seeing an outdated version of `CHANGELOG.md`. The current version looks like this: https://github.com/moq/moq4/blob/9ea157cf61f1eb7deeadc419cac08d1aa9fd4f44/CHANGELOG.md.

Reply:
My bad, rebased and updated.